Demo a pull request of JW-update_to merging into main (#1)
davidelliott2/git_training: practice_scripts/introduce_yourself.md

# Introduce yourself

> Hello, my name is David and I work for the UK Health Security Agency.

Author: Don't think we should be sharing our employer, might draw unwanted attention 🕶️ 
```suggestion
Hello, my name is David and I work for a government agency.
```
